src/index.ts

```diff
@@ -1099,6 +1184,36 @@ async function main() {
           run(() => mergeNeurons(args.sourceNeuronId, args.targetNeuronId))
         )
     )
+    .addCommand(
+      new Command("register-vote")
+        .description("Vote on a specific proposal.")
+        .requiredOption("--neuron-id <neuron-id>", "Neuron ID", tryParseBigInt)
+        .requiredOption(
+          "--proposal-id <proposal-id>",
+          "Proposal ID",
+          tryParseBigInt
+        )
+        .requiredOption("--vote <vote>", "Vote (1 - YES, 2 - NO)", tryParseInt)
+        .action((args) =>
+          run(() => registerVote(args.neuronId, args.proposalId, args.vote))
+        )
+    )
+    .addCommand(
+      new Command("set-followees")
+        .description(
+          "Set followees of a neuron in a specific topic. This will overwrite the existing followees for that topic."
+        )
+        .requiredOption("--neuron-id <neuron-id>", "Neuron ID", tryParseBigInt)
+        .requiredOption("--topic <topic>", "Topic", tryParseInt)
```

Comment:
I would not be sure if I should pass the topic as text or id. worth adding the keyword `"Topic ID"`?

Reply:
Yes, good point!